app-main/.editorconfig

```diff
@@ -1,12 +0,0 @@
-# EditorConfig helps developers define and maintain consistent
```

Comment:
We don't want to remove the editorconfig. If you can add this back in that would be great :)